TenantFile/Api/Controllers/ImageController.cs

```diff
@@ -36,7 +40,13 @@ public async Task<ActionResult<ImageListResult>> Get()
             var names = objects.Select(x => x.Name);
             return Ok(new ImageListResult { Images = names });
         }
-
+        
+        [HttpGet("/api/images/{Id}")]
+        public async Task<ActionResult<ImageListResult>> GetTenantPhotos(string id)
+        {
+            return Ok(await _documentDb.GetImagesById(id));
```

Comment:
It looks like this might be able to replace GetImage? Or does this have a different purpose?